recipes/longhorn.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
```

Comment:
Let's comment and add installation reference URL and short description

Reply:
added description